### PR TITLE
DHFPROD-5897: Clearing date range facet does not clear facet.

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx [hunk 1]
```diff
@@ -514,5 +514,30 @@ describe('Verify numeric facet can be applied', () => {
...
+      cy.visit('/');
+      cy.contains(Application.title);
+      cy.loginAsTestUserWithRoles("pii-reader","hub-central-developer").withRequest();
+      cy.waitUntil(() => toolbar.getExploreToolbarIcon()).click();
+      cy.waitUntil(() => browsePage.getExploreButton()).click();
+      browsePage.waitForSpinnerToDisappear();
+      browsePage.waitForTableToLoad();
+  });
+
+  it('Verify clearing date range facet clears corresponding selected facet', () => {
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx:530]: This test case should be added to an existing test in order to have our e2e tests more like real user scenarios.

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx]: Merged with existing test. Thanks.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx [hunk 2]
```diff
@@ -513,6 +513,17 @@ describe('Verify numeric facet can be applied', () => {
         browsePage.getFacetApplyButton().click();
         browsePage.getRangeFacet(3024).should('exist');
         browsePage.getClearAllButton().should('exist');
+        browsePage.getClearAllButton().click();
+        //Verify clearing date range facet clears corresponding selected facet
+        browsePage.selectEntity('Customer');
```
Comment (Sanjeevani19) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx:518]: @timur-isangulov once we use clearAllButton the page will come back to it's original state right ? Did you have any reason to repeat all the statements from 518-520 ?

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx]: Removed duplicate lines. Thanks.

**Review by brucean52 — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**